static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -5,383 +5,331 @@ import {
-    display: 'flex',
-    flexDirection: 'column',
-
-    [mobile]: {
-        alignItems: 'baseline',
-        flexDirection: 'row',
-        paddingTop: 0,
-        ul: {
-            display: 'flex',
-        },
-        li: {
-            marginLeft: '30px',
-        },
-    },
-    [tablet]: {
-        borderLeft: '1px solid #767676',
-        alignItems: 'baseline',
-        flexDirection: 'row',
-        paddingTop: 0,
-        ul: {
-            display: 'flex',
-        },
-        li: {
-            marginLeft: '30px',
-        },
-    },
-    b: {
-        fontSize: '16px',
-        fontFamily: '"Guardian Egyptian Web",Georgia,serif',
-        fontWeight: '900',
-    },
-    ul: {
-        fontFamily: 'Guardian Text Sans Web',
-        fontSize: '14px',
-    },
-    a: {
-        color: '#cbcbcb', // TODO: add to the palette
-    },
-    li: {
-        marginTop: '12px',
-    },
-    img: {
-        marginRight: '6px',
-        verticalAlign: 'middle',
-        height: '18px',
-    },
-});
-
-const Button = styled('button')(({ isPlay, pillarColor }) => ({
-    background: 'none',
-    border: 0,
-    cursor: 'pointer',
-    margin: 0,
-    ':focus': {
-        outline: 'none', // ಠ_ಠ
-    },
-    padding: isPlay ? '0 45px' : 0,
-    svg: {
-        width: isPlay ? '60px' : '26px',
-        height: isPlay ? '60px' : '26px',
-    },
-    circle: {
-        fill: pillarColor,
-        stroke: pillarColor,
-    },
 
     [leftCol]: {
-        padding: isPlay ? '0 12px' : 0,
-        svg: {
-            width: isPlay ? '50px' : '24px',
-            height: isPlay ? '50px' : '24px',
-        },
+        position: 'absolute',
+        bottom: '0',
+        left: '0',
+        marginBottom: '9px',
     },
-
-    [wide]: {
-        padding: isPlay ? '0 20px' : 0,
-        svg: {
-            width: isPlay ? '74px' : '26px',
-            height: isPlay ? '74px' : '26px',
-        },
-    },
-}));
+});
 
 type Props = {
     sourceUrl: string,
     mediaId: string,
     downloadUrl: string,
-    iTunesUrl: string,
-    pillar: string,
 };
 
 type State = {
     ready: boolean,
     playing: boolean,
+    muted: boolean,
+    scrubbing: boolean,
     currentTime: number,
     duration: number,
-    volume: number,
-    bins: ?NodeList<HTMLElement>,
-    interval: number,
     currentOffset: number,
+    currentOffsetPx: number,
     hasBeenPlayed: boolean,
+    waveWidthPx: number,
 };
 
-export default class AudioPlayer extends Component<Props, State> {
+export class AudioPlayer extends Component<Props, State> {
     constructor(props: Props) {
         super(props);
         this.state = {
             ready: false,
             playing: false,
+            muted: false,
+            scrubbing: false,
             currentTime: 0,
-            duration: NaN,
-            volume: NaN,
-            bins: null,
-            interval: NaN,
             currentOffset: 0,
+            currentOffsetPx: 0,
+            duration: 3000,
```

Comment:
Should probably initialise this at 0? (think i added this to test something out)